src/sets.jl

```diff
@@ -1252,6 +1252,38 @@ function Base.:(==)(x::CountAtLeast, y::CountAtLeast)
     return x.dimension == y.dimension && x.n == y.n && x.set == y.set
 end
 
+"""
+    CountGreaterThan(dimension::Int)
+
+The set ``\\{(c, y, x) \\in \\mathbb{R}^{1+1+d}\\}`` such that `c` is strictly
```

Comment:
So there's a decision whether we match MOI's epigraph-variable-first order, or MiniZinc's `x, y, c` order.

Reply:
Discussed with @blegat.